site/tryit/README.md

```diff
@@ -20,6 +20,10 @@ docker run \
 sorintlab/agolademo \
 serve --embedded-etcd --components all-base,executor
 ```
+where:
+   * `-v=path/to/local/agola/data` is your agola data directory
```

Comment:
The option has a missing piece (it should be `-v /path/to/gitea-data:/data`). Also, if we really have to explain this, we should write something like

`-v /path/to/gitea-data:/data will mount a local `/path/to/gitea-data` directory (be sure to change it) to /data inside the container`